command/server/server_command.go

```diff
@@ -143,13 +143,36 @@ func (c *ServerCommand) DefineFlags() {
 		FlagOptional: true,
 	}
 
-	c.FlagMap["max-peers"] = helper.FlagDescriptor{
-		Description: fmt.Sprintf("Sets the client's max peer count. Default: %d", helper.DefaultConfig().Network.MaxPeers),
+	c.FlagMap["max-inbound-peers"] = helper.FlagDescriptor{
+		Description: "Sets the client's maximum number of inbound peers allowded. Default: 40",
 		Arguments: []string{
 			"PEER_COUNT",
 		},
 		FlagOptional: true,
 	}
+	c.FlagMap["max-outbound-peers"] = helper.FlagDescriptor{
+		Description: "Sets the client's max no.of outbound peers allowded. Default: 10",
+		Arguments: []string{
+			"PEER_COUNT",
+		},
+		FlagOptional: true,
+	}
+
+	c.FlagMap["locals"] = helper.FlagDescriptor{
+		Description: "Sets comma separated accounts whose transactions are treated as locals",
+		Arguments: []string{
+			"LOCALS",
+		},
+		FlagOptional: true,
+	}
+
+	c.FlagMap["nolocals"] = helper.FlagDescriptor{
+		Description: "Sets flag to disable price exemptions for locally submitted transactions",
+		Arguments: []string{
+			"NOLOCALS",
+		},
+		FlagOptional: true,
+	}
```

Comment:
`--locals` and `--nolocals` has been removed in TxPool v2 PR.

Reply:
Cleaned up commit-[link](https://github.com/0xPolygon/polygon-edge/pull/343/commits/62ece3a00a9b352650e95900bff12c1c57516486)